io/net/virt-net/network_virtualization.py

```diff
@@ -295,6 +295,44 @@ def test_backingdevadd(self):
             self.fail("Failed to add backing device")
         self.check_dmesg_error()
 
+    def test_disable_enable_dev(self):
+        '''
+        Test if disabling and enabling of an adapter works
+        '''
+        self.disable_enable_dev('d')
+        self.is_disabled_enabled_dev('1')
+        self.disable_enable_dev('e')
+        self.is_disabled_enabled_dev('0')
+
```

Comment:
Normally at the end of each test we collect and check for dmsg, will you be adding self.check_dmesg_error()